x-pack/plugins/lens/common/expressions/heatmap_chart/heatmap_legend.ts

```diff
@@ -19,6 +19,14 @@ export interface HeatmapLegendConfig {
    * Position of the legend relative to the chart
    */
   position: Position;
+  /**
+   * Defines the number of lines per legend item
+   */
+  maxLines?: number;
+  /**
+   * Defines the number of lines per legend item
+   */
+  truncate?: boolean;
```

Comment:
nit: maybe we could change it to 
```suggestion
  shouldTruncate?: boolean;
```
so it's obvious is a boolean and not a function (verb specifies it's a function)

Reply:
Very good observation :)